.github/workflows/publish.yml

```diff
@@ -31,6 +31,7 @@ jobs:
         run: |
           git config user.name "Kirby Bot"
           git config user.email "<>"
+          git remote add gh-origin https://kirbycibot:${secrets.VERSION_BUMP_GITHUB_TOKEN}@github.com/kirbydesign/designsystem.git
```

Comment:
We might need to change the push commands to be git push gh-origin master

Reply:
Add via token to checkout action instead